### ISSUE TITLE
Exit Code 66

### ISSUE THREAD

**Issue by tylermrolfe** [sxg/ForecastIO#41]
Receiving this error after running 'carthage update'
I've also seen other resolved issues in the carthage repo that require you to remove the CarthageKit folder from Library/Cache but I've already tried that.

Cartfile: 
`github "sxg/ForecastIO" "swift3"`

```
A shell task (/usr/bin/xcrun xcodebuild -workspace /Users/tylermrolfe/Developer/Fake Name/Carthage/Checkouts/ForecastIO/ForecastIO.xcworkspace -scheme "Pods-ForecastIO OS X Tests-ForecastIO" -configuration Release CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES clean -showBuildSettings) failed with exit code 66:
xcodebuild: error: Scheme Pods-ForecastIO OS X Tests-ForecastIO is not currently configured for the clean action.
```

**Comment by sxg** [comment]
I believe the issue has to do with the renaming of OS X to macOS. The ForecastIO OS X scheme was renamed to ForecastIO macOS. It seems that Carthage (or maybe CocoaPods) is using the old name in building the project. I'm not a big Carthage user, so I'll have to spend a little more time looking into this issue. Otherwise, I'm actively working on a new Swift 3 compatible version of ForecastIO. It's currently in a stable state (according to Travis CI) on the [dev branch](https://github.com/sxg/ForecastIO/tree/dev). Replace `swift3` with `dev` in your `Cartfile`, and it should work. I'll be formally releasing the new version of ForecastIO within the next few days if you can wait for that. Let me know if this helps!

**Comment by CharlesVu** [comment]
I still have this error when trying to do a carthage update on my project. I tried the master and dev branches

> $ carthage update --platform iOS
> 
> xcodebuild: error: Scheme Pods-ForecastIO tvOS Tests-ForecastIO is not currently configured for the build action.

**Comment by sxg** [comment]
Hey, sorry for the very delayed response. The issue seems to be with the ForecastIO arm scheme in the main Xcode workspace. I've deleted it and released a new version (3.0.3) of ForecastIO. This fixed the issue for me. You should be able to add `github "sxg/ForecastIO"` in your `Cartfile` to make it work. Just run `carthage update` like usual. Let me know if this resolves your issue.
